test/lstm_common.hpp

```diff
@@ -2928,6 +2930,10 @@ struct lstm_basic_driver : test_driver
     {
 
 #if(MIOPEN_BACKEND_OPENCL == 1)
+#if WORKAROUND_ISSUE_692 == 1
```

Comment:
[Note] FYI `#if WORKAROUND_ISSUE_692` is just enough. No need to fix this in this PR.